actool/discover.go

```diff
@@ -25,7 +25,7 @@ func init() {
 
 func runDiscover(args []string) (exit int) {
 	if len(args) < 1 {
-		fmt.Fprintf(os.Stderr, "discover: at least one name required")
+		fmt.Fprintf(os.Stderr, "discover: at least one name required\n")
```

Comment:
Thanks for the fix. Would you mind actually just changing this to use the [`stderr` helper function](https://github.com/appc/spec/blob/master/actool/actool.go#L99) for consistency with elsehwere?